source/ios/AdaptiveCards/ADCIOSFramework/ADCIOSFramework/ACRColumnRenderer.mm

```diff
@@ -6,48 +6,46 @@
 //
 
 #import "ACRColumnRenderer.h"
+#import "ACRColumnView.h"
 #import "ACRRegistration.h"
 #import "Column.h"
 #import "SharedAdaptiveCard.h"
 
 @implementation ACRColumnRenderer
 
-+ (ACRColumnRenderer* ) getInstance
```

Comment:
The latest one is the one that will be used. 

---
In reply to: [128563718](https://github.com/Microsoft/AdaptiveCards/pull/488#discussion_r128563718) [](ancestors = 128563718,128559390)